internal/querycoordv2/task/merger.go

```diff
@@ -120,6 +120,8 @@ func (merger *Merger[K, R]) merge(id K, queue chan MergeableTask[K, R]) {
 	go merger.mergeQueue(id, queue)
 }
 
+const maxSegmentsPerTask = 3
```

Comment:
move it to configuration ?

Reply:
what does this impact? and what factors we need to think of if trying to update it